### ISSUE TITLE
Google Chrome Lighthouse Report

### ISSUE THREAD

**Issue by mannyreds11** [rollbar/rollbar.js#476]
We recently ran a google chrome lighthouse performance test against our website and noticed that we had 2 failed Audits related to rollbar (see attachment) with the following message:

"Does not use passive listeners to improve scrolling performance
Consider marking your touch and wheel event listeners as `passive` to improve your page's scroll performance”

More information related to this issue: https://developers.google.com/web/tools/lighthouse/audits/passive-event-listeners

We're still using version 2.2.3. Is this something that has been looked at or fixed in a most current version?

![screen shot 2017-10-24 at 4 07 58 pm](https://user-images.githubusercontent.com/1425097/31968544-16373ab8-b8d7-11e7-842c-6d15b80f8637.png)

**Comment by brianr** [comment]
Interesting. As far as I can tell, rollbar.js doesn't add any wheel or touch event listeners directly (the ones listed on the page you linked)... however, it does wrap your own calls to addEventListener.

Questions:

- does your app add any event listeners for `wheel`, `mousewheel`, `touchstart`, or `touchmove`?
- are you passing `{passive: true}` in those calls?
- if you remove rollbar.js and run the test again, do you see the failed audits coming from somewhere else?

**Comment by mannyreds11** [comment]
@brianr Thanks for the quick response. We have a 3rd party `mousewheel` event in our `commons.js` file I was able to modify and got rid of the error. Still not sure why it was reporting as coming from `rollbar.min.js`. This may belong to a different issue but could it be the `autoInstrument` feature? I noticed our `console.log`s  were all showing `rollbar.min.js` on the right (see attachment) and I had to set `autoInstrument: { log: false }` to fix it.

before:
![screen shot 2017-10-25 at 2 18 59 pm](https://user-images.githubusercontent.com/1425097/32019371-9b6655b4-b992-11e7-89d5-9956079930f0.png)


after:
![screen shot 2017-10-25 at 2 41 22 pm](https://user-images.githubusercontent.com/1425097/32019379-a0476ec4-b992-11e7-89a0-82d8fa21697f.png)
